src/components/common/Badge.tsx

```
@@ -0,0 +1,110 @@
+          {props.children}
+          {suffix}
+        </span>
+      );
+
+    // 제출 상태 배지
+    case 'submission':
+      const {label, icon} = SubmissionMeta[props.status!];
+
+      return (
+        <span
+          className={submissionBadgeStyles({
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)